### ISSUE TITLE
Contact Page Redesign

### ISSUE THREAD

**Issue by avi-11** [dsckiet/website#53]
# Overview
 
 Redesign the contacts page on the website.

# Description

 DSC KIET website is an official website made with HTML, CSS, and JAVASCRIPT. This website describes the organization's workflow, events, and members.

# Expected Outcomes

 A nice, simple-looking UI/UX design for the contacts page.

# Skills

 UI/UX designing, FIGMA, ADOBE XD, CANVA.

# Possible Mentors

 [Avaneesh Singh](https://github.com/avi-11)

# Expected Time

 2 days

# Difficulty

 easy

**Comment by avi-11** [comment]
# Use This Proposal Template as a Reference to write your own proposal for Msoc

# Your Issue Name

### Your Background and Technical Skills.
<hr>

### What problem are you trying to solve and Your Approach?
<hr>

###  Your Timeline to solve the problem.
<hr>

### Your Commitments.
<hr>

---

**Comment by avi-11** [comment]
# Example of Proposal.


# Improve the generated PDF in the PDF/LaTeX backend

### About me
I have been programming since High School and I have been Advocating for Open-Source projects for the past 2 years. I
have experienced in Python , JavaScript, and Dart[Flutter] and I am currently learning Julia. I organized events on Open-Source in my College and I am the founding member of an Open-Source community [fossc](https://www.saynotocp.org/) where I mentor Students in their Open-Source Journey. For more information about me You can head over to my [website](https://kanishkpachauri.netlify.app/)
<hr>

### Issue - Intermittent CI failures with PDF builds 
Issue - The "PDF/LaTeX backend" stage often fails on CI, causing the docs not to deploy (since they only deploy if that stage passes) Restarting the build usually fixes it though.

## Proposed Solution….
● Fixing any bug on the CI end.
● Debug any issue in the test of the LaTeX backend
● Rewriting guided Tests for it so that it can be fixed throughout the process.
<hr>

###  TimeLine
## May 20 - June 12
Community bonding, understanding Documenter.jl Codebase, re-scoping project implementation/timeline with help from mentors.

## June 13 - June 23
Learning Julia Essentials and other essentials for integration and fixing of issue #1196 and #1467 and #1342 .

## June 24 - July 15
Adding the Proposed Solution for the issue #1196 and #1467

<hr>

### Commitments.
● I’ll try my best to learn everything which is essential for the
project.
● I’ll Create regular Blog posts on the progress of the project
● I’ll try my best to complete the project on time.
● In case of any change in the above timeline, I will inform
the mentors in advance.
<hr>

---

**Comment by Hackintosh-02** [comment]
> # CONTACT PAGE REDESIGN 
> ### Your Background and Technical Skills.
I am Ashutosh Sharma I am relatively new to Web Designing with around 4 months of experience and I want to start my open source contributions, I have worked on designs on Figma and think I can resolve this issue.
> ### What problem are you trying to solve and Your Approach?
I am trying to solve the issue of redesigning the contact page, I will change error of slack written in place of discord, I will change the design of the page and will make it more attractive.
> ### Your Timeline to solve the problem.
I am going to start on 11th September and the project will be finished around 13th September.
> ### Your Commitments.
I am committed to giving my all in order to solve this issue ASAP.

**Comment by Jhalakupadhyay** [comment]
CONTACT PAGE REDESIGN.

YOUR BACKGROUND AND TECHNICAL SKILLS.

I am jhalak upadhyay , I have been doing website and app designing since last SIX MONTHS and I had made a design for one of my own app and . I think I can add to the project very well.

WHAT PROBLEMS ARE YOU TRYING TO SOLVE AND YOUR APPROACH.

I will try to add more aesthetic look in the  design of the contacts page of the website and try my best to give it a better look.

YOUR TIMELINE TO SOLVE THE PROBLEM.

I will start the problem by 11th of  September and finish it by 12th of  September 9:00 pm.

YOUR COMMITMENTS.

I will try use the best of my creativity to add a new look to the page , try to make it look more aesthetic. I have already started to work on it.

**Comment by Jhalakupadhyay** [comment]
> # Use This Proposal Template as a Reference to write your own proposal for Msoc
> # CONTACT PAGE REDESIGN
> ### Your Background and Technical Skills.
I am jhalak upadhyay , I have been doing website and app designing since last SIX MONTHS and I had made a design for one of my own app and . I think I can add to the project very well.

> ### What problem are you trying to solve and Your Approach?
I will try to add more aesthetic look in the  design of the contacts page of the website and try my best to give it a better look.
> ### Your Timeline to solve the problem.
I will start the problem by 11th of  September and finish it by 12th of  September 9:00 pm.
> ### Your Commitments.
I will try use the best of my creativity to add a new look to the page , try to make it look more aesthetic. I have already started to work on it.
